### DISCUSSION TITLE
Sublime Text Support

### DISCUSSION THREAD

**Discussion by bitsper2nd** [artempyanykh/marksman#7] [Ideas]
Hi, I was looking for a Markdown Language Sever and stumbled upon your zeta-note. My idea is to add markdown support to the Sublime Text LSP plugin. I would appreciate[ any help](https://forum.sublimetext.com/t/making-lsp-md/61797/3). I already started making the plugin. I will be using [this one](https://github.com/sublimelsp/LSP-TexLab) as a base.

**Comment by artempyanykh** [comment]
That's a great initiative! 💖 
I don't use Sublime Text, so not sure what sort of help I can offer though.

**Reply by bitsper2nd** [reply]
A [plugin has been made](https://github.com/bitsper2nd/LSP-marksman) for Sublime Text.

**Reply by artempyanykh** [reply]
@bitsper2nd wow, congrats! 🥳 Would you like to create a PR mentioning Sublime Text's integration in [Marksman's README](https://github.com/artempyanykh/marksman/blob/main/README.md)?

**Reply by bitsper2nd** [reply]
Just add the following:

* Sublime Text 
  * via [Sublime-LSP](https://github.com/bitsper2nd/LSP-marksman) (automatic server installation).

**Reply by bitsper2nd** [reply]
So I talked to the devs behind Sublime LSP and they adopted my plugin. So [the new URL/repo is this one](https://github.com/sublimelsp/LSP-marksman). Hopefully soon it will be available in Sublime Text package control too.

**Reply by artempyanykh** [reply]
That's awesome! I'll update the README shortly.

**Reply by bitsper2nd** [reply]
LSP-marksman is finally on [Package Control for Sublime Text](https://packagecontrol.io/packages/LSP-marksman). This the equivalent of the VS Code Extensions Marketplace.
